catalog/ui/src/app/components/ServiceNamespaceSelect.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useMemo, useState } from 'react';
+import useSWR, { useSWRConfig } from 'swr';
+import useSession from '@app/utils/useSession';
+import { ContextSelector, ContextSelectorItem } from '@patternfly/react-core';
+import { ServiceNamespace } from '@app/types';
+import { apiPaths, fetcher } from '@app/api';
+
+import './service-namespace-select.css';
+
+const ServiceNamespaceSelect: React.FC<{
+  allowSelectAll: boolean;
+  currentNamespaceName: string;
+  isPlain: boolean;
+  isText: boolean;
+  serviceNamespaces: ServiceNamespace[];
```

Comment:
not needed, is defined inside the component

Reply:
Removed.